source/_integrations/knx.markdown

```diff
@@ -1768,6 +1771,63 @@ entity_category:
   default: None
 {% endconfiguration %}
 
+## Time
+
+The KNX time platform allows to send time values to the KNX bus and update its state from received telegrams. It can optionally respond to read requests from the KNX bus with its current state.
+
+<div class='note'>
+
+Time entities without a `state_address` will restore their last known state after Home Assistant was restarted.
+
+Times having a `state_address` configured request their current state from the KNX bus.
+
+</div>
+
+<div class='note'>
+
+The `day` filed of the time telegram will always be set to 0 (`no day`).
+
+</div>
+
+```yaml
+# Example configuration.yaml entry
+knx:
+  time:
+    - name: "Time"
+      address: "0/0/2"
+      state_address: "0/0/2"
+```
+
```

Comment:
state_address: "0/0/2"

are these zeroes the `day`fields of the time telegram, as mentioned above? Or is this unrelated? How do I know the address?

Reply:
"x/y/z" is the most common notation for KNX group addresses. KNX users know this very well 😉
The day field is always present in a DPT 10.001 telegram (its the first 3 bit of the 3 byte payload) and can't be configured.